workspace/test_injection.sh

```
@@ -0,0 +1,100 @@
+fi
+
+cp project.godot "$BACKUP_FILE"
+
+echo "🗑️ Wiping Windows .godot cache to force a clean Linux build..."
+rm -rf .godot/
+rm -rf export/web/*
+mkdir -p export/web
+
+echo "⚙️ Injecting secret using safe ENVIRON AWK script..."
+# 1. Escape for Godot's parser and EXPORT to environment
+export GODOT_ESCAPED=$(printf '%s' "$PRODUCTION_SALT" | sed 's/\\/\\\\/g; s/"/\\"/g')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)